### DISCUSSION TITLE
ask for help

### DISCUSSION THREAD

**Discussion by vpn777china** [strongswan/strongswan#1272] [Q&A]
I roughly understood How to Set Up an IKEv2 VPN Server with StrongSwan. Go through the tutorial linked below.
https://www.digitalocean.com/community/tutorials/how-to-set-up-an-ikev2-vpn-server-with-strongswan-on-ubuntu-22-04
----------------
But I want to ask a question. The tutorial in the link only explains how to configure 1 VPN server. What should I do if I want the "generated certificate" to configure more VPN servers?
![截屏2022-09-03 16 28 47](https://user-images.githubusercontent.com/98731060/188262788-13b71cba-fd99-49a0-a014-280d6d77c866.png)
![截屏2022-09-03 16 28 59](https://user-images.githubusercontent.com/98731060/188262794-754dfe74-e6ef-4692-9afd-7842b0fe753f.png)
I am a user from China. I hope the "generated certificate" can help more Chinese break through the Great Firewall of China. So, if I want to help 1000 people get over the wall using this method, how do I configure StrongSwan with multiple VPN servers?

**Comment by Thermi** [comment]
IKE/IPsec is not capable of covertly penetrating through the GFW because of its unique protocol signature.
Depending on how you configure the whole setup you might be capable of generating one certificate and using it on all, or having to use distinct ones per server. It entirely depends on the client, because it depends on the ID the client sends and expects the server to present.

**Reply by vpn777china** [reply]
Thank you for your answer,
The tutorials I've seen are all one certificate for one server. Would love to find a tutorial on "Multiple VPS using the same CA root certificate".
If you know a tutorial on this, please give me a hint.
Anyway, thank you very much.

**Reply by Thermi** [reply]
you just issue several certificates with different DNs, from different private keys, and with different SANs.

**Reply by Thermi** [reply]
Just a hint: Try using shadowsocks. That is probably stealthier than IPsec.

**Reply by vpn777china** [reply]
See a tutorial: resolve the ip of multiple VPS servers to the second-level domain name of the same domain name. It can realize that multiple VPN servers share one CA certificate. 
I am ready to give it a try.
